docs/development.md

```diff
@@ -63,6 +63,25 @@ bundle exec foreman start
 
 When using the dev server on Firefox you need to accept the self-signed certificate. You should go to `https://centos7-devel.example.com:3808` (or your equivalent) and add an exception for this certificate. Chrome does not throw any warnings if you have accepted the same certificate already for `https://centos7-devel.example.com`.
 
+### Reviewing Pull Requests
+
+It's easy to checkout pull requests from projects that were installed in development environment. All projects are cloned in vagrant's home, e.g. /home/vagrant/foreman, /home/vagrant/katello etc. In order to apply some PR to any of this project, you can use a reviewer script. See following example
```

Comment:
Note that this documentation and the code in the role itself lock this down to `/home/vagrant` pathing. This can differ per deployment due to the configuration options provided by `puppet-katello_devel`. A user can modify the deployment directory from the default (https://github.com/theforeman/puppet-katello_devel/blob/c76572d8752cd4198b972ac4b4fa5a834412e403/manifests/params.pp#L10).

Reply:
yes, but currently there are other places where vagrant was hardcoded - https://github.com/theforeman/forklift/blob/master/roles/foreman_proxy_content/tasks/devel_install.yml#L5 so I took it as a given fact

I can definitely change it but it says e.g., should I modify it to `~/foreman` and `~/katello`?

Reply:
That sounds good